package.json

```diff
@@ -30,11 +30,11 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^4.0.7",
-    "@theforeman/eslint-plugin-foreman": "^4.0.7",
-    "@theforeman/stories": "^4.0.7",
-    "@theforeman/test": "^4.0.7",
-    "@theforeman/vendor-dev": "^4.0.7",
+    "@theforeman/builder": "^4.2.0",
+    "@theforeman/eslint-plugin-foreman": "^4.2.0",
+    "@theforeman/stories": "^4.2.0",
+    "@theforeman/test": "^4.2.0",
+    "@theforeman/vendor-dev": "^4.2.0",
```

Comment:
@sharvit I opened up https://github.com/theforeman/foreman/pull/7590 - but I didn't see the versions were updated here. Should these be `^5.0.0` now that that is released?